topics/basics/plugin_structure/plugin_configuration_file.md

```diff
@@ -13,71 +13,94 @@ Allowed HTML elements include text formatting, paragraphs, and lists.
-       Don't repeat the name of the plugin.
-       For plugins that add language/platform/framework support, the description MUST specify
-       the version of the corresponding language/platform/framework.
-       Don't mention the IDE compatibility. E.g., don't say "Adds support to IntelliJ IDEA for..."
-       Displayed in the "Plugins" settings dialog and the plugin repository Web interface.
-       Simple HTML elements can be included between <![CDATA[  ]]> tags.  -->
-  <description>Integrates Volume Snapshot Service W10</description>
-
-  <!-- Description of changes in the latest version of the plugin.
-       Displayed in the "Plugins" settings dialog and the plugin repository Web interface.
-       Simple HTML elements can be included between <![CDATA[  ]]> tags.  -->
-  <change-notes>Initial release of the plugin.</change-notes>
+  <id>org.jetbrains.plugins.template</id>
 
-  <!-- Plugin version
-       Recommended format is BRANCH.BUILD.FIX (MAJOR.MINOR.FIX)
-       Displayed in the "Plugins" settings dialog and the plugin repository Web interface.  -->
+  <!-- Public plugin name should be written in Title Case. Guidelines: -->
+  <!-- TODO: [URL HERE] -->
+  <name>Plugin Template</name>
+
+  <!-- Plugin version. It is recommended to use the SemVer approach: https://semver.org
+     Displayed in the "Plugins" settings dialog and the plugin repository Web interface. -->
   <version>1.0.0</version>
 
-  <!-- The vendor of the plugin.
-       The optional "url" attribute specifies the URL of the vendor homepage.
-       The optional "email" attribute specifies the e-mail address of the vendor.
-       Displayed in the "Plugins" settings dialog and the plugin repository Web interface. -->
-  <vendor url="https://www.company.com" email="support@company.com">A Company Inc.</vendor>
-
-  <!-- Mandatory dependencies on plugins or modules.
-       The FQN module names in <depends> elements are used to determine IDE compatibility for the plugin.
-       Include at least the module shown below to indicate compatibility with IntelliJ Platform-based products.
-       Also, include dependencies on other plugins as needed.
-       See "Compatibility with Multiple Products" and "Plugin Dependencies" for more information.  -->
+  <!-- A displayed Vendor name or Organization ID (if you have one created. The optional `URL` attribute specifies
+       the link to the vendor’s homepage.The optional `email` attribute specifies the vendor’s e-mail address.
+       Displayed on the Plugins Page. -->
+  <vendor url="https://plugins.jetbrains.com" email="marketplace@jetbrains.com">JetBrains</vendor>
+
+  <!-- If you decide to make your plugin paid, you will need to define the parameters in the <product-descriptor> tag.
+       You can also enable free functionality in a paid plugin. Learn more in a guide to selling plugin:
+       https://plugins.jetbrains.com/build-and-market -->
```

Comment:
please clarify in uppercase letters this tag must _not_ be used at all for free plugins